src/test_utils/mod.rs

```diff
@@ -50,7 +50,9 @@ pub fn create_rng() -> RAND {
 // Reads the json test files
 pub fn json_reader(file_name: &str) -> BufReader<File> {
     let mut file_path_buf = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
-    file_path_buf.push("src/test_utils/hash_to_curve_vectors/");
+    file_path_buf.push("src");
```

Comment:
any reasons?

Reply:
This commit is from upstream, maybe cause by one version late sync.